base/stream.jl

```diff
@@ -595,6 +597,7 @@ function _uv_hook_close(t::Timer)
     unpreserve_handle(t)
     disassociate_julia_struct(t)
     t.handle = C_NULL
+    notify(t.cond)
```

Comment:
this fix is the cause of the warning in the `repl.jl` test: the Timer callback on line 100 of test/repl.jl is supposed to accept an argument `t`